automation/submission-perf-inc.sh

```diff
@@ -143,7 +147,13 @@ launchSubmission() {
         optionalEntityNameField="\"entityName\":\"${entityName}\","
     fi
 
-    curl \
+    response=$(curl \
```

Comment:
This curl command is the most liable to fail, from what I've seen. It's also the only one submitting a non-trivial amount of data. Giving it some bonus retry attempt/time